addons/website/views/snippets/s_text_cover.xml

```diff
@@ -2,15 +2,19 @@
 <odoo>
 
 <template id="s_text_cover" name="Text Cover">
-    <section class="s_text_cover">
+    <section class="s_text_cover o_cc o_cc2">
         <div class="container-fluid">
-            <div class="row">
-                <div class="o_colored_level o_cc o_cc4 col-lg-5 pt160 pb160 px-5">
+            <div class="row o_grid_mode" data-row-count="12">
+                <div class="o_grid_item g-height-10 g-col-lg-7 col-lg-7 o_colored_level o_cc o_cc1" style="z-index: 1; grid-area: 2 / 3 / 11 / 8; --grid-item-padding-x: 24px; --grid-item-padding-y: 24px;">
                     <h1 class="display-3">Sell Online. <br/>Easily.</h1>
-                    <p class="lead">Write one or two paragraphs describing your product, services or a specific feature. To be successful your content needs to be useful to your readers.</p>
-                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Contact us</t></a>
+                    <br/>
+                    <p class="lead">Sell online easily with a user-friendly platform that streamlines all the steps,<br/>including setup, inventory management, and payment processing.</p>
```

Comment:
Remove the `<br>` or finetune its visibility for very large screens only, eg: `<br class="d-none d-xxl-inline"/>`
```suggestion
                    <p class="lead">Sell online easily with a user-friendly platform that streamlines all the steps, including setup, inventory management, and payment processing.</p>
```
| this PR | expected result |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/8b940c24-e8bd-44b8-9b5d-cc6bce88c0ae) | ![image](https://github.com/user-attachments/assets/2eb514ff-8519-4807-b4d0-dfed468d18f4) |

Reply:
Edit: actually the `br` seems not necessary... Do you confirm?

Reply:
Shouldn't be here indeed... Nice catch